pyupgrade.py

```diff
@@ -981,6 +983,12 @@ def _is_six(self, node, names):
             node.attr in names
         )
 
+    def _is_utf8_codec(self, encoding):
```

Comment:
eheh not exactly what I meant -- since it doesn't make sense as a method (doesn't use `self`)

Reply:
I can move it